lineapy/api/api.py

```diff
@@ -236,8 +234,7 @@ def _try_write_to_db(value: object) -> str:
         # Don't see an easy way to catch all possible exceptions from the to_pickle, so just catch everything for now
         logger.error(e)
         track(ExceptionEvent(ErrorType.SAVE, "Pickling error"))
-        raise e
-    return artifact_filename
+        raise ArtifactSaveException() from e
```

Comment:
did this change need any update to existing tests?

Reply:
The tests pass without making this change. I still believe that raising any random exception from `lineapy.save` is a bad idea. This change is also in line with db.py line 146, which raises an `ArtifactSaveException` from a generic exception.